README.md

```diff
@@ -0,0 +1,94 @@
+##############################################################################################
+##############################################################################################
+#POT GAME							JACKPOT GAME							JACKPOT GAME	 #
+##############################################################################################
+##############################################################################################
+
+
+
+##############################################################################################
+#									Installation											 #
+##############################################################################################
+
+1. Python3
+
+
+
+##############################################################################################
+#									HOW TO RUN?												 #
+##############################################################################################
+
+1. Add the path where you download this app in main.py file.
+2. To run the sample test case, enter the following command in terminal -> python3 main.py
+3. To add test-cases, please follow the sample test case method.
+4. For adding more slots, please refer to folder wise documentation stated below.
+
+
+
+
+##############################################################################################
+#									DOCUMENTATION											 #
+##############################################################################################
+
+
+#################
+#    MODELS		#
+#################
+
+1. Game:
```

Comment:
Game should be a Service and not a model. 
There could be a model called GameConfiguration instead with properties like num_slots and max_tries.